libraries/AP_HAL_ChibiOS/hwdef/F4BY_H743/README.md

```diff
@@ -0,0 +1,129 @@
+# F4BY_H743 Flight Controller with STM32H743VIT MCU
+
+The F4BY_H743 flight controller shop: https://f4by.com/en/?order/our_product
+The instructions, schematic, 3D model  are available here: https://f4by.com/en/?doc/fc_f4by_v3.0.1_h743
+
+## Features
+    Processor
+        STM32H743VIT6 MCU (2M flash)
+    Sensors
+        ICM-42688-P IMU (F4BY_H743 v3.0.3)  
+        IIM-42652   IMU (F4BY_H743 v3.0.2)
+        MPU6000     IMU (F4BY_H743 v3.0.1)   
+        MS5611 Barometer 
+        QMC5883P Compass   
+    Power
+        power from external power module (range +5.0 ... +6.0V)
+        anslog current and voltage analog inputs (range 0 ... +6,6V)
+    Interfaces
+        12x PWM outputs DShot capable 
+        1x RC input
+        5x UARTs/serial for GPS, bi-directional RC and other peripherals)
+        1x I2C port for external compass, airspeed, etc.
+        1x CAN bus
+        Buzzer output
+        microSDCard for logging, Lua scripts etc.
+        USB type-C port
+        USB GHS type port for external connector.
+        DFU bootloader button
+
+
+## Analog sensors
+
+- Analog airspeed PC0 (0...+3.3v range) pin 10
```

Comment:
analog airspeed and rssi are usually given separate sections in readme as they are in future wiki page and gives the params required to use it